packages/field-plugin/src/createFieldPlugin/FieldPluginData.ts

```diff
@@ -8,10 +8,24 @@ export type FieldPluginData<Content> = {
   content: Content
   options: Record<string, string>
   spaceId: number | undefined
+  interfaceLang: string
```

Comment:
Since we have `storyLang`, I named it `interfaceLang`, but any other suggestion is welcome.

Reply:
`storyfront` is our project name for the frontend, and it's not publicly used. So I picked `interface` 🤔